package-lock.json

```diff
@@ -0,0 +1,6467 @@
+{
```

Comment:
Please use `yarn` instead of `npm`. There is already a yarn lockfile.

Reply:
Sorry I'm not sure what you meant by that, did you mean I should delete the *package-lock.json* file?

Reply:
Yes please! Sorry, I thought you also ended up installing new packages as well. But it looks like you weren't.

Reply:
Or apparently, I can?